modules/installation-configuration-parameters.adoc

```diff
@@ -192,7 +192,12 @@ Only IPv4 addresses are supported.
 endif::agent,bare,ibm-power,ibm-z,vsphere[]
 
 ifdef::agent,bare,ibm-power,ibm-z,vsphere[]
+Consider the following information before you configure network parameters for your cluster:
+
 * If you use the {openshift-networking} OVN-Kubernetes network plugin, both IPv4 and IPv6 address families are supported.
+* If your infrastructure uses IPv6 as the primary network layer protocol in a Dynamic Host Configuration Protocol (DHCP) server and your cluster does not support IPv6, you can use the Kubernetes NMState Operator to disable IPv6 on your infrastructure.  
```

Comment:
I'm not sure you can use the operator for this, I think it needs to be a day 1 config. Otherwise the nodes will initially come up with ipv6 enabled and may cause problems.

Given that we're starting to support dual stack on non-baremetal platforms, maybe we should not specify a method at all since it may vary depending on platform. We could shorten this to: "If your infrastructure uses IPv6 in a Dynamic Host Configuration Protocol (DHCP) server and your cluster does not support IPv6, you can disable IPv6 on your nodes' interfaces."

Oh, and as I'm reading this closer, it doesn't matter if v6 is primary or not so I think we should drop that as well. The updated line above includes all of the changes I would suggest here.

Reply:
Updated. 

I'm not sure if "Oh, and as I'm reading this closer, it doesn't matter if v6 is primary or not so I think we should drop that as well. The updated line above includes all of the changes I would suggest here." indicates I should remove one of the bullet points or an IPv6 reference?